underfs/abfs/src/main/java/alluxio/underfs/abfs/AbfsUnderFileSystem.java

```diff
@@ -66,8 +67,28 @@ private static Configuration createAbfsConfiguration(UnderFileSystemConfiguratio
             conf.getString(PropertyKey.ABFS_CLIENT_ID));
         abfsConf.set(PropertyKey.ABFS_CLIENT_SECRET.getName(),
             conf.getString(PropertyKey.ABFS_CLIENT_SECRET));
+        clientCredentials = true;
       }
     }
+
+    if (!clientCredentials && !sharedKey) {
```

Comment:
so the following code is an alternative scenario of the way with that `clientCredentials == true`?
they will not happen at the same time?

Reply:
yes, they are different auth type.